themes/digital.gov/src/js/guide-mobile-menu.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Display horizontal desktop or vertical mobile menu when viewport size changes
+ *
+ */
+
+const dgGuideMenuBar = document.querySelector(".dg-guide__menu-bar");
+const dgAccordionMobileMenu = document.querySelector(".guide-mobile-menu");
+const dgAccordionContent = document.querySelector(
+  ".guide-mobile-menu .usa-accordion__content"
+);
+const desktopBreakpoint = window.matchMedia("(min-width: 480px)");
+
+/**
+ *  Reset menu based on screen size
+ *
+ *  @param {MediaQueryList} breakpoint
+ */
+function handleResize(breakpoint) {
+  if (breakpoint.matches) {
+    dgAccordionMobileMenu.classList.add("guide-mobile-menu--hide");
+    dgGuideMenuBar.classList.remove("dg-guide__menu-bar--hide");
+  } else {
+    dgAccordionMobileMenu.classList.remove("guide-mobile-menu--hide");
+    dgGuideMenuBar.classList.add("dg-guide__menu-bar--hide");
+  }
+}
+
+/**
+ * Update usa-accordion content to hidden on page load
+ */
+document.addEventListener("DOMContentLoaded", () => {
+  dgAccordionContent.setAttribute("hidden", true);
+});
+
+/**
+ * Initialize guide functions for toggling guide menu
+ */
+function initGuideMobileMenu() {
+  handleResize(desktopBreakpoint);
+  desktopBreakpoint.addEventListener("change", handleResize);
+}
+
+initGuideMobileMenu();
```

Comment:
Because this is bundled it will init on all pages. Consider a check to see if `dgGuideMenuBar` exists before running any of the code.

Check console on homepage:
![image](https://github.com/GSA/digitalgov.gov/assets/3385219/b6c840c1-39bf-4598-96f2-df60c03b34c0)

Reply:
Great point, wrapped in a conditional